### ISSUE TITLE
Tour not coded according to best practice

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14744]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

A modal dialog appears after login until the user has clicked “Don’t show this tour again”. Looks nice and friendly, but for a screen reader user to even find this information he or she must navigate to the very end of the page. 

### Specifics

According to best practice for modal dialogs, they should capture the full focus of the user until dismissed. Users shouldn’t even be able to tab to or read the content behind the modal until it has been closed, and of course the modal – just as all other groups of content, should have a label. Please see (and implement) https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/ 

<img width="309" alt="image" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/9b13c779-3b34-4f67-8fa3-0d1652b72d8e">

### Steps to reproduce

Log in to Umbraco 10.4.0 for the first time. If the "Tour invite" appears, try to find it with a screen reader. Try also to navigate to and from it with the keyboard. 

### Expected result / actual result

See specifics, above.

**Comment by andr317c** [comment]
Hey!

Thanks for taking the time to raise this issue! This is something we would like help with, I'll set this issue as up for grabs 😄

**Comment by nagolucky18** [comment]
I will take a look at this one.
